eng/common/instructions/azsdk-tools/typespec-to-sdk.instructions.md

```diff
@@ -88,19 +88,29 @@ Your goal is to guide user through the process of generating SDKs from TypeSpec
 **Actions**:
 1. Run `GetSDKPullRequestDetails` to fetch generated SDK PR info.
 
-## Step 9: Create release plan
+## Step 9: Validate Label and Codeowners
```

Comment:
Files in `eng/common` go in a separate PR in tools repo. https://github.com/Azure/azure-sdk-tools/blob/main/doc/common/common_engsys.md#updating